shared-bindings/is31fl3741/__init__.c

```diff
@@ -24,16 +24,59 @@
  * THE SOFTWARE.
  */
 
+#include "shared-bindings/is31fl3741/__init__.h"
+
 #include <stdint.h>
+#include <stdbool.h>
 
 #include "py/obj.h"
 #include "py/runtime.h"
 
+#include "shared-bindings/busio/I2C.h"
 #include "shared-bindings/is31fl3741/IS31FL3741.h"
 
+//| def is31fl3741_write(i2c: busio.I2C, addr: int, mapping: Tuple[int, ...], buf: ReadableBuffer) -> None:
```

Comment:
I think this should be moved into the IS31FL3741 object. Providing the same i2c bus and device id to three objects in the lib seems weird. I'll post a suggestion to a library example.